src/Microsoft/Silk.NET.Direct2D/Structs/D2D1Device2VtblExtensions.gen.cs

```diff
@@ -170,198 +170,6 @@ public static unsafe int CreatePrintControl(this ComPtr<ID2D1Device2> thisVtbl,
         return ret;
     }
 
-    /// <summary>To be documented.</summary>
-    public static unsafe int CreatePrintControl(this ComPtr<ID2D1Device2> thisVtbl, IWICImagingFactory* wicFactory, ref IPrintDocumentPackageTarget documentTarget, [Flow(Silk.NET.Core.Native.FlowDirection.In)] PrintControlProperties* printControlProperties, ID2D1PrintControl** printControl)
```

Comment:
This is fine as this is new code but worth looking at. IPrintDocumentPackageTarget is forward declared, should we add documenttarget.h to Win32Extras?

Reply:
heres a patch file with a WIP fix for this, i shall continue it tomorrow, as it is late for me, unless you want to pick it up and fix the last few errors (a few missing structs still and simple BuildTools patch is needed)
[wip.patch](https://github.com/dotnet/Silk.NET/files/11137133/wip.patch)

Reply:
i think this should have been fixed by the latest commits?